.scripts/pre-commit

```diff
@@ -1,30 +1,24 @@
 #!/bin/bash
 
-GREEN='\033[0;32m'
-RED='\033[0;31m'
-NO_COLOR='\033[0m'
+RED=$(printf '\033[0;31m')
+CLEAR=$(printf '\033[0m')
```

Comment:
consider rename to no color or white or something like that

Reply:
Actually I see it referred to by 'reset' most often around the web. Are you cool with that?

Reply:
ok